spec/services/escort_creator_spec.rb

```diff
@@ -146,6 +164,15 @@ def expect_offences_to_be_cloned(existent_escort, new_escort)
     expect(offences_attributes).to match_array(expected_offences_attributes)
   end
 
+  def expect_move_special_vehicle_details_to_be_cloned(existent_escort, new_escort)
+    %i[require_special_vehicle require_special_vehicle_details
```

Comment:
To be consistent with the rest of the spec, I would add this attributes to the method:
```
except_move_attributes
```